recipes-ni/ni-base-system-image-tests/files/fs_permissions_known.py

```diff
@@ -0,0 +1,162 @@
+import argparse
+import os
+import sys
+import hashlib
+import stat
+import pwd
+import grp
+import fs_permissions_shared
+
+# Function to return tree instead of variable since it contains functions inside its structure
+def known_permissions_tree():
+    return {
```

Comment:
Seems we lost some things. [Here](https://github.com/ni/meta-nilrt/commit/ffcbb3c73cbaf95dbd1c5146d84df7ffb2b38835)'s the last state of the CSV:

```
Path,Mode,User,Group,Link Target
/boot/bootmode,-rw-r--r--,admin,administrators,
/boot/grub/!(recoverytool-ni-version),-rw-rw-r--,admin,ni,
/boot/grub/recoverytool-ni-version,-r--r--r--,admin,administrators,
/boot/runmode/!(bzImage),-rw-r--r--,admin,administrators,
/boot/runmode/bzImage,lrwxrwxrwx,admin,administrators,bzImage-$(uname -r)
/etc/fstab,-rw-r--r--,admin,administrators,
/home/admin,drwxr-xr-x,admin,administrators,
/home/lvuser,drwxrwsr-x,lvuser,ni,
/home/root,lrwxrwxrwx,admin,administrators,/home/admin
/home/webserv,drwxr-sr-x,webserv,ni,
/lib/modules/$(uname -r)/!(build|kernel|source),-rw-r--r--,admin,administrators,
/lib/modules/$(uname -r)/@(build|kernel),drwxr-xr-x,admin,administrators,
/lib/modules/$(uname -r)/source,lrwxrwxrwx,admin,administrators,build
/tmp,lrwxrwxrwx,admin,administrators,/var/tmp
```

Reply:
I've readded the missing fstab and home files